meetings/2023-02-01.md

```diff
@@ -50,36 +57,125 @@ URL: https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-fe
+* SC: Let's start with the ACTION and get people to discuss. 
+* SC: I think people should be given an option of making a chat in their storage and send a copy to the central storage. Web Annotation has something where there is ??? I do this in dokieli. When you write an annotation, if you have a storage you can decide to put the copy of that in multiple places. If a document offers a shared ??? service and you have a storage you can use both or only one. I think the problem is not different for Solid chat but it depends on how these apps are designed. Whether to use one storage or offer people to write to their storage. Some of the problems are solved from other communities. 
 
 ### Add Solid QA
 URL: https://github.com/solid/specification/pull/495
 
 * SC: Any comments here? Objections to merge?
-
-
+* TBL: Where does this come from? 
+* SC: Part of the test suite panel work. Pete and Michiel are co-authors and we have open meetings. Also based on W3C QA work. 
+* SC: Preview of Solid QA: https://htmlpreview.github.io/?https://github.com/solid/specification/blob/53cbcefa5a93c444ad40798d01f0fa1287e4d3d8/ED/qa.html
+* SC: We're treating this as a spec in that we have some expectations from ??? so that we have a uniform approach .
+* TBL: How dow e know it won't be too bureacratic? 
+* SC: I don't know. There is confusion as to what qualifies as a correct test.  These are some rules as to what the reports should look like, what the test suite needs, etc. It's a work in progress. But the specification tests that Pete wrote does conform to most of this. 
+* TBL: There should be a condensed ??? so I don't have to read through the document. 
+* SC: Comments or question on the direction we're going with this?
+* TBL: If this is what test suite folks are happy with, can you get someone to review it? 
+* SC: Both approved. 
+* TBL: It needs an abstract which summarizes what it describes, but I support. 
+* SC: Is this your review before merging or something you want to add to the PR? 
+* eP: This morning I joined a call with people working with [Keycloak](https://www.keycloak.org/). Justin and JaneiroDigital are working on adding Solid-OIDC and were asking about test suite. It'd be interesting to work with them, someone who is not a solid insider to implement a solid spec and use test suite to provide the reports. Good opportunity to get feedback. 
+* SC: People are welcome to join the CG or if any existing members want to collaborate with them that's fine. 
+* eP: I'd expect that'll be Justin and maybe I will help. 
+* SC: Ok. I won't merge now. 
+* TBL: There's a person on the W3C team. He couldn't go to the ??? conference. He sat and read all the abstracts and said "now I don't have to go to the ??? conference" because he relied on the fact that the abstract can save you ??? if you have more time you can read through. Maybe you can get ChatGPT's eyes on it.
```

Comment:
```suggestion
* TBL: There's a person on the W3C team. He couldn't go to the (TPAC?) ??? conference. He sat and read all the abstracts and said "now I don't have to go to the (TPAC?) ??? conference" because he relied on the fact that the abstract can save you ??? If you have more time, you can read through. Maybe you can get ChatGPT's eyes on it.
```